modules/nw-annotating-a-route-with-a-cookie-name.adoc

```diff
@@ -9,32 +9,54 @@
 [id="nw-annotating-a-route-with-a-cookie-name_{context}"]
 = Annotating a route with a cookie
 
-You can set a cookie name to overwrite the default, auto-generated one for the
-route. This allows the application receiving route traffic to know the cookie
-name. By deleting the cookie it can force the next request to re-choose an
-endpoint. So, if a server was overloaded it tries to remove the requests from the
-client and redistribute them.
+You can set a cookie name to overwrite the default, auto-generated one for the route. This allows the application receiving route traffic to know the cookie name. By deleting the cookie it can force the next request to re-choose an endpoint. So, if a server was overloaded it tries to remove the requests from the client and redistribute them.
 
 .Procedure
 
-. Annotate the route with the desired cookie name:
+. Annotate the route with the specified cookie name:
 +
 [source,terminal]
 ----
-$ oc annotate route <route_name> router.openshift.io/<cookie_name>="-<cookie_annotation>"
+$ oc annotate route <route_name> router.openshift.io/cookie_name="<cookie_name>"
 ----
 +
-For example, to annotate the cookie name of `my_cookie` to the `my_route` with
-the annotation of `my_cookie_annotation`:
+--
+where:
+
+`<route_name>`:: Specifies the name of the route.
+`<cookie_name>`:: Specifies the name for the cookie.
+--
++
+For example, to annotate the route `my_route` with the cookie name `my_cookie`:
++
+[source,terminal]
+----
+$ oc annotate route my_route router.openshift.io/cookie_name="my_cookie"
+----
+
+. Capture the route host name in a variable:
 +
 [source,terminal]
 ----
-$ oc annotate route my_route router.openshift.io/my_cookie="-my_cookie_annotation"
+$ ROUTE_NAME=$(oc get route <route_name> -o jsonpath='{.spec.host}')
 ----
++
+--
+where:
+
+`<route_name>`:: Specifies the name of the route.
+--
 
-. Save the cookie, and access the route:
+. Save the cookie, and then access the route:
++
+[source,terminal]
+----
+$ curl $ROUTE_NAME -k -c /tmp/my_cookie
```

Comment:
So "my_cookie" and "my_route" were used in an example in step 1, but it looks like in step 2 you're back to using <route_name>, so here in step 3, should it be using <cookie_name> in both of these code blocks?

Like `$ curl $ROUTE_NAME -k -c /tmp/<cookie_name>` and `$ curl $ROUTE_NAME -k -b /tmp/<cookie_name>` ?

Reply:
In this example, `/tmp/my_cookie` is a file that saves the cookie(s) sent by the server. To prevent confusion, should the file be renamed to something like `saved_cookie`?

Reply:
Ah, so it's not related to the "my_cookie" example from step 1? If so, then yeah I'd suggest another name to avoid confusion.

Reply:
I changed the filename to `cookie_jar` to avoid confusion.